larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -232,103 +229,107 @@ public int runScenarios(IbisContext ibisContext, String paramLogLevel,
 		config.setUseLogBuffer(false);
 		debugMessage("Start debugging to out");
 		debugMessage("Execute scenario(s) if execute parameter present and scenarios root directory did not change");
+		if (paramExecute == null) {
+			config.flushWriters();
+			return 0;
+		}
 		int scenariosFailed = 0;
-		if (paramExecute != null) {
```

Comment:
I did not remove all these lines, but took them out the big `if (paramExecute != null)`, but that messes up the diff. Can't help it.